verilog/formatting/align.cc

```diff
@@ -317,6 +374,13 @@ static const verible::AlignedFormattingHandler kActualNamedPortAligner{
         AlignmentCellScannerGenerator<ActualNamedPortColumnSchemaScanner>(),
 };
 
+static const verible::AlignedFormattingHandler kBlockItemStatementListAligner{
+    .extract_alignment_groups = &verible::GetSubpartitionsBetweenBlankLines,
```

Comment:
I suspect this will need to add logic to step around non-assignment statements, and only select blocks of consecutive assignments.